FreeRTOS/FreeRTOS
# Updates FreeRTOS-Plus-Trace  to a submodule pointer

Updates FreeRTOS-Plus-Trace to a submodule pointer

Description
-----------

Updates FreeRTOS-Plus-Trace  to a submodule pointer and the relevant demo's make file to support the new design
Percepio recommends using ring buffer for 64 bit system and hence updates the to support that behavior.

Test Steps
-----------


Checklist:
----------



Related Issue
-----------


Testing
----------
Build and run the following demos and see if you can generate Trace.dump file:
- FreeRTOS_Plus_CLI_with_Trace_Windows_Simulator
- FreeRTOS_Plus_TCP_Echo_Posix
- WIN32-MSVC-Static-Allocation-Only
- Posix_GCC
- WIN32-MSVC
- WIN32-MingW



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
